pkg/util/resource/resource.go

```diff
@@ -45,6 +49,16 @@ func GetPodResourceRequests(podSpec *corev1.PodSpec) corev1.ResourceList {
 	reqs := make(corev1.ResourceList)
 
 	for _, container := range podSpec.Containers {
+		// GPUs are only supposed to be specified in the limits section.
+		// Therefore, when only GPU is specified in the limits section,
+		// we need to obtain the resource request of GPU from limits.
+		// See details in https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/.
```

Comment:
Should we extend this logic to all resource types?

Reply:
I think it'ok.